test/witan/send/acceptance/workspace_test.clj

```diff
@@ -59,6 +60,15 @@
 
   :splice-ncy - sets a national curriculum year to ignore transitions of prior to :filter-transitions-from year (optional)"
   [{:keys [iterations output? transition-modifier transitions-file modify-transitions-from filter-transitions-from splice-ncy]}]
+  (report/reset-send-report)
+  (report/info (str "Input Data: " (clojure.string/replace inputs-path #"/" " ")))
+  (report/info (str "Number of iterations: " iterations))
+  (report/info (str "Output charts produced: " output?))
+  (report/info (str "Transitions modifier: " (if (nil? transition-modifier) "None" transition-modifier)))
+  (report/info (str "Transitions file: " (if (nil? transitions-file) "None" transitions-file)))
+  (report/info (str "Modify transitions from: " (if (nil? modify-transitions-from) "None" modify-transitions-from)))
+  (report/info (str "Filter transitions from: " (if (nil? filter-transitions-from) "None" filter-transitions-from)))
+  (report/info (str "Splice NCY: " (if (nil? splice-ncy) "None" splice-ncy) "\n"))
```

Comment:
It would be nice if `report/info` was multi-arity and joined the args with spaces (this is what most loggers, including Timbre do).